DOWNLOAD_STATS.md

```diff
@@ -5,14 +5,30 @@
 | `@nolyfill/array-buffer-byte-length` | [![npm](https://img.shields.io/npm/dt/@nolyfill/array-buffer-byte-length.svg?style=flat-square&logo=npm&logoColor=white&label=total%20downloads&color=333)](https://www.npmjs.com/package/@nolyfill/array-buffer-byte-length) |
 | `@nolyfill/array-includes` | [![npm](https://img.shields.io/npm/dt/@nolyfill/array-includes.svg?style=flat-square&logo=npm&logoColor=white&label=total%20downloads&color=333)](https://www.npmjs.com/package/@nolyfill/array-includes) |
 | `@nolyfill/array.from` | [![npm](https://img.shields.io/npm/dt/@nolyfill/array.from.svg?style=flat-square&logo=npm&logoColor=white&label=total%20downloads&color=333)](https://www.npmjs.com/package/@nolyfill/array.from) |
+| `@nolyfill/array.of` | [![npm](https://img.shields.io/npm/dt/@nolyfill/array.of.svg?style=flat-square&logo=npm&logoColor=white&label=total%20downloads&color=333)](https://www.npmjs.com/package/@nolyfill/array.of) |
 | `@nolyfill/array.prototype.at` | [![npm](https://img.shields.io/npm/dt/@nolyfill/array.prototype.at.svg?style=flat-square&logo=npm&logoColor=white&label=total%20downloads&color=333)](https://www.npmjs.com/package/@nolyfill/array.prototype.at) |
+| `@nolyfill/array.prototype.concat` | [![npm](https://img.shields.io/npm/dt/@nolyfill/array.prototype.concat.svg?style=flat-square&logo=npm&logoColor=white&label=total%20downloads&color=333)](https://www.npmjs.com/package/@nolyfill/array.prototype.concat) |
```

Comment:
Nobody uses `array.prototype.concat` on npm. It has 0 dependent and its weekly downloads are less than 1000.